assets/js/modules/adsense/datastore/adunits.js

```diff
@@ -0,0 +1,126 @@
+/**
+ * `modules/adsense` data store: Ad Units.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import invariant from 'invariant';
+
+/**
+ * Internal dependencies
+ */
+import API from 'googlesitekit-api';
+import Data from 'googlesitekit-data';
+import { STORE_NAME } from './constants';
+import { createFetchStore } from '../../../googlesitekit/data/create-fetch-store';
+
+const fetchGetAdUnitsStore = createFetchStore( {
+	baseName: 'getAdUnits',
+	controlCallback: ( { accountID, clientID } ) => {
+		return API.get( 'modules', 'adsense', 'adunits', { accountID, clientID }, {
+			useCache: false,
+		} );
+	},
+	reducerCallback: ( state, adunits, { accountID, clientID } ) => {
+		return {
+			...state,
+			adunits: {
+				...state.adunits,
+				[ `${ accountID }::${ clientID }` ]: [ ...adunits.items ],
```

Comment:
Looks like we don't need destructing here and can simply assign items:

```suggestion
				[ `${ accountID }::${ clientID }` ]: adunits.items,
```